src/config.rs

```diff
@@ -14,8 +51,254 @@ impl Config {
     }
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Clone, Debug)]
+pub struct LocalizedConfig {
```

Comment:
why LocalizedConfig?

Reply:
To combine a config with the location where it was found, hence it was "localized". I feel this is confusing with localization in terms of l10n too, but couldn't come up with a better name. Suggestions welcome!

Reply:
I guess we can leave it for now. I suspect once we support multiple configs, it may become `Configs` or something plural